<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hauntedHouse</class>
 <widget class="QMainWindow" name="hauntedHouse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>hauntedHouse</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="gameWindow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>400</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/imgs/resources/ZorkBackground.png);</string>
     </property>
     <widget class="QPushButton" name="goToNextTab">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/Start.png);
background-color: rgb(128, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="quitButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/Quit.png);
background-color: rgb(128, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(86, 86, 86);</string>
     </property>
     <widget class="QGraphicsView" name="gameGraphics">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>740</width>
        <height>270</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="UpButton">
      <property name="geometry">
       <rect>
        <x>619</x>
        <y>280</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/ic_arrow_upward_black_48dp_1x.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="DownButton">
      <property name="geometry">
       <rect>
        <x>619</x>
        <y>345</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/ic_arrow_downward_black_48dp_1x.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="RightButton">
      <property name="geometry">
       <rect>
        <x>678</x>
        <y>310</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/ic_arrow_forward_black_48dp_1x.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="LeftButton">
      <property name="geometry">
       <rect>
        <x>558</x>
        <y>310</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/ic_arrow_back_black_48dp_1x.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="gameConsoleBox">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>285</y>
        <width>375</width>
        <height>100</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);
font: 12pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Hello &lt;/h1&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="FlashlightButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>290</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/imgs/resources/Flashlight.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>UpButton</zorder>
     <zorder>DownButton</zorder>
     <zorder>LeftButton</zorder>
     <zorder>gameConsoleBox</zorder>
     <zorder>gameGraphics</zorder>
     <zorder>FlashlightButton</zorder>
     <zorder>RightButton</zorder>
    </widget>
    <widget class="QWidget" name="gameOver">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
image: url(:/imgs/resources/Bonus/the end.png);</string>
     </property>
     <widget class="QPushButton" name="playAgain">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>750</width>
        <height>400</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
